src/lib/types/model.ts

```diff
@@ -365,3 +365,66 @@ export interface IFeatureStrategySegment {
     featureStrategyId: string;
     segmentId: number;
 }
+
+export interface ISuggestChangeSet {
+    id: number;
+    state: string;
+    project: string;
+    environment: string;
+    createdBy?: string;
+    createdAt?: Date;
+    changes?: ISuggestChange[];
+    events?: ISuggestChangeEvent[];
+    // approvalFlow: IChangeSetApprovalFlow;
+}
+export interface IApprovalFlow {}
+
+export interface IChangeSetApprovalFlow {
+    id: string;
+    flow: IApprovalFlow;
+}
+
+export interface ISuggestChange {
+    id: number;
+    action: SuggestChangeAction;
+    feature: string;
+    payload?: unknown;
+    createdBy?: string;
```

Comment:
Is it linked with `User`? How do we store info about who created it, because UI has user name and avatar. Maybe some metadata on this can solve it.